packages/deployer/test/sample-project/contracts/modules/UpgradeModule.sol

```diff
@@ -0,0 +1,106 @@
+//SPDX-License-Identifier: Unlicense
+pragma solidity ^0.8.0;
+
+import "../mixins/OwnerMixin.sol";
+import "../storage/ProxyStorage.sol";
+import "../libraries/Address.sol";
+
+contract UpgradeModule is ProxyStorageNamespace, OwnerMixin {
+    /* MUTATIVE FUNCTIONS */
+
+    function upgradeTo(address newImplementation) public onlyOwnerOrProxy {
```

Comment:
For sample project, just allow bricking with a small comment with a warning.

In the future, we may want to use this anti-bricking mechanism and/or investigate OpenZeppelin's mechanism in their official UUPS proxy implementation.